src/pages/320x240x16/trim_page.c

```diff
@@ -97,20 +97,20 @@ void PAGE_TrimEditInit(int page)
     };
 
     //Row 1
-    GUI_CreateLabelBox(&gui_ed->srclbl, COL1, ROW1, COL2-COL1, 0, &LABEL_FONT, NULL, NULL, _tr("Input"));
+    GUI_CreateLabelBox(&gui_ed->srclbl, COL1, ROW1, COL2-COL1, 0, &LABEL_FONT, GUI_Localize, NULL, _tr_noop("Input"));
     GUI_CreateTextSource(&gui_ed->src, COL2, ROW1, TEXTSELECT_96, NULL, set_source_cb, set_input_source_cb, &tp->trim.src);
     //Row 2
-    GUI_CreateLabelBox(&gui_ed->steplbl, COL1, ROW2, COL2-COL1, 0, &LABEL_FONT, NULL, NULL, _tr("Trim Step"));
+    GUI_CreateLabelBox(&gui_ed->steplbl, COL1, ROW2, COL2-COL1, 0, &LABEL_FONT, GUI_Localize, NULL, _tr_noop("Trim Step"));
     GUI_CreateTextSelect(&gui_ed->step, COL2, ROW2, TEXTSELECT_96, NULL,
                          set_trimstep_cb, (void *)(long)(tp->index + 0x100)); //0x100: Use tp->trim
     //Row 3
-    GUI_CreateLabelBox(&gui_ed->neglbl, COL1, ROW3, COL2-COL1, 0, &LABEL_FONT, NULL, NULL, _tr("Trim -"));
+    GUI_CreateLabelBox(&gui_ed->neglbl, COL1, ROW3, COL2-COL1, 0, &LABEL_FONT, GUI_Localize, NULL, _tr_noop("Trim -"));
     GUI_CreateTextSelect(&gui_ed->neg, COL2, ROW3, TEXTSELECT_96, NULL, set_trim_cb, &tp->trim.neg);
     //Row 4
```

Comment:
Lint Error:
```
105:  At least two spaces is best between code and comments  
105:  Should have a space between // and comment  
105:  Use int16/int64/etc, rather than the C type long  
106:  Should have a space between // and comment  
109:  Should have a space between // and comment  
```